third_party/typings/cordova/inappbrowser.d.ts

```diff
@@ -0,0 +1,7 @@
+interface Window {
```

Comment:
I think would be help to provide a link to Cordova's documentation for `InAppBrowser`, if there is one.